alerts/openshift-virtualization-operator/NodeStatusMaxImagesExceeded.md

```diff
@@ -5,7 +5,7 @@
 This alert fires when a node exceeds the configured maximum number of images
 reported in the node status (default: 50). Once this limit is reached, the
 scheduler might not be able to accurately schedule new pods because the reported
-image count is wrong.
+image count is wrong [[1]](#references).
```

Comment:
I recommend you use a simple link to BZ instead of adding a reference section:
`[BZ#1984442](https://bugzilla.redhat.com/1984442)`. The link should be one space AFTER the period:

image count is wrong. [BZetc]

Reply:
Done.

Reply:
Thanks for letting me know, I fixed it.